bodhi/server/models.py

```diff
@@ -34,7 +34,7 @@
 
 
 from sqlalchemy import Unicode, UnicodeText, Integer, Boolean
-from sqlalchemy import DateTime, engine_from_config
+from sqlalchemy import DateTime
```

Comment:
We shouldn't be making very many modifications to this file, other than merging the Enum items in. This import is needed, and I see a lot of other unrelated changes elsewhere in the file.